packetbeat/etc/beat.yml

```diff
@@ -21,6 +21,20 @@ protocols:
     # Enable ICMPv4 and ICMPv6 monitoring. Default: false
     enabled: true
 
+  amqp:
+    # Configure the ports where to listen for AMQP traffic. You can disable
+    # the AMQP protocol by commenting out the list of ports.
+    ports: [5672]
+    # uncomment the following to truncate messages that are published and avoid
+    # huge messages being indexed. Default limit size is 1000 bytes.
+    #max_body_length: 100
+    # uncomment the following if you want to hide the header fields
+    # in header frames.
+    #parse_headers: false
+    # uncomment the following if you want to hide the additional arguments
+    # of method frames.
+    #parse_arguments: false
```

Comment:
In config file all commented out options do have default values. For consistency use default values instead of asking user to comment out this line.